lib/bemhtml/index.js

```diff
@@ -118,7 +118,7 @@ BEMHTML.prototype.render = function(context, entity, tag, js, bem, cls, mix,
 
 var OPTIONAL_END_TAGS = {
   // html4 https://html.spec.whatwg.org/multipage/syntax.html#optional-tags
```

Comment:
I think we should change that link to https://www.w3.org/TR/html4/index/elements.html